.github/workflows/eks.yaml

```diff
@@ -107,9 +107,12 @@ jobs:
         run: |
           eksctl create nodegroup \
             --cluster ${{ env.clusterName }} \
+            --nodes 2 \
+            --instance-types "t3.medium,t3a.medium" \
```

Comment:
Hmm, GKE machines seem to be configured with 2vCPUs and 8GB of ram. These ones are 2vCPUs and 4GB of ram. we should have consistency across all cloud providers to disregard potential flakes caused by resource constrains.

Reply:
Actually it's GKE that's an outlier: they don't have a machine type offering 2vCPUs/4GB, which is also the combo provided by `Standard_B2s` on AKS.

We can switch to `Standard_B2ms` and `t3.large` variants though if we want 2vCPUs/8GB everywhere, it just costs a bit more.

Reply:
in GKE we can have custom machine types. Something like `--instance-types "custom-2-4096"` give us a 2vCPU with a 4GB memory.

Reply:
Good call, I didn't think about using the custom machine types. Changing GKE.